modules/serverless-rn-1-16-0.adoc

```diff
@@ -0,0 +1,116 @@
+[id="serverless-rn-1-16-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.16.0
+
+[id="new-features-1-16-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.22.0.
+* {ServerlessProductName} now uses Knative Eventing 0.22.0.
+* {ServerlessProductName} now uses Kourier 0.22.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.22.0.
+* {ServerlessProductName} now uses Knative Kafka 0.22.0.
+* The `kn func` CLI plugin now uses `func` 0.16.0.
+* The `kn func emit` command has been added to the functions `kn` plugin. This command allows you to send events to test locally deployed functions.
+
+[id="fixed-issues-1-16-0_{context}"]
+== Fixed issues
+
+[id="known-issues-1-16-0_{context}"]
+== Known issues
+
+// Add a note about the following to the mTLS docs when they're ready to merge
+* If Service Mesh is enabled with mTLS, metrics for Knative Serving are disabled by default because Service Mesh prevents Prometheus from scraping metrics.
++
+If you want to enable Knative Serving metrics for use with Service Mesh and mTLS, you must complete the following steps:
+
+.. Specify `prometheus` as the `metrics.backend-destination` in the `observability` spec of the Knative Serving custom resource (CR):
++
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+spec:
+  config:
+    observability:
+      metrics.backend-destination: "prometheus"
+----
++
+This step prevents metrics from being disabled by default.
+
+.. Apply the following network policy to allow traffic from the Prometheus namespace:
++
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-from-openshift-monitoring-ns
+  namespace: knative-serving
+spec:
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          name: "openshift-monitoring"
+  podSelector: {}
+  policyTypes:
+  - Ingress
+----
+
+.. Modify and reapply the default Service Mesh control plane in the `istio-system` namespace, so that it includes the following spec:
++
+[source,yaml]
+----
+spec:
+  proxy:
+    networking:
+      trafficControl:
+        inbound:
+          excludedPorts:
+          - 8444
+----
+
+* If you deploy Service Mesh CRs with the Istio ingress enabled, you may see the following warning in the `istio-ingressgateway` pod:
```

Comment:
> !  may verb
Use only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility. "May" is overused in technical information. In most cases, use "can" or "might" instead.

This one reads like "might." And I see at least one other instance of "may," so please check the rest of the PR and make updates.